src/main/java/nextstep/subway/applicaion/StationService.java

```diff
@@ -22,24 +22,17 @@ public StationService(StationRepository stationRepository) {
     @Transactional
     public StationResponse saveStation(StationRequest stationRequest) {
         Station station = stationRepository.save(new Station(stationRequest.getName()));
-        return createStationResponse(station);
+        return StationResponse.of(station);
     }
 
     public List<StationResponse> findAllStations() {
         return stationRepository.findAll().stream()
-                .map(this::createStationResponse)
+                .map(station -> StationResponse.of(station))
```

Comment:
여기서도 method reference로 작성해 주셔도 좋을 것 같아요 😃 
```suggestion
                .map(StationResponse::of)
```